force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -40,17 +40,23 @@ public with sharing class ClassesObjectsMethods {
      */
     public static List<String> practiceStringMethods() {
         // Initialize a String with 'Hello, Salesforce!'
-        
-        // Get the length of the String        
+        String myString = 'Hello, Salesforce!';
+        // Get the length of the String
+        String strLength = String.valueOf(myString.length());        
         // Get a substring of the String of the first 5 characters
+        String substring = myString.substring(0, 5);
         // Convert the String to lower case
-        
+        String strLowCase = myString.toLowerCase();
         // Return the list with the string data
+        List<String> stringData = new List<String>();
         // In the first position add the string length
+        stringData.add(strLength);
         // In the second position add the substring
+        stringData.add(substring);
         // In the third position add the string in lower case
+        stringData.add(strLowCase);
 
-        return null; // Replace null with the variable you used to store the result
+        return stringData;
```

Comment:
Good work. The only thing I would do differently is that rather than creating variables you don't really need. I would assign to the List directly. 

```
public static List<String> practiceStringMethods() {
        List<String> strings = new List<String>();
        String helloSalesforce = 'Hello, Salesforce!';
        strings.add(String.valueOf(helloSalesforce.length()));
        strings.add(helloSalesforce.substring(0, 5));
        strings.add(helloSalesforce.toLowerCase());
        return strings;
    }
    ```